Sources/CgRPC/shim/channel.c

```diff
@@ -63,6 +63,22 @@ cgrpc_channel *cgrpc_channel_create_secure(const char *address,
   return c;
 }
 
+cgrpc_channel *cgrpc_channel_create_google(const char *address,
+                                           grpc_arg *args,
+                                           int num_args) {
+    cgrpc_channel *c = (cgrpc_channel *) malloc(sizeof (cgrpc_channel));
```

Comment:
Nit: Please use two spaces for indentation. `make project` will set up your project file for that.